websites/R/Radio Garden/presence.ts

```diff
@@ -1,58 +1,64 @@
 const presence = new Presence({
-	clientId: "687070418804408445", //The client ID of the Application created at https://discordapp.com/developers/applications
-});
+		clientId: "687070418804408445", //The client ID of the Application created at https://discordapp.com/developers/applications
+	}),
+	elapsed = Math.floor(Date.now() / 1000);
 
 let details: string, state: string, currentState: string;
 
 presence.on("UpdateData", async () => {
 	if (!document.title.includes("Explore live radio by rotating")) {
 		details =
-			document.querySelector("._title_165lg_36")?.textContent ??
-			"Unknown Radio"; // which radio we're tuning right now
+			document
+				.querySelector('[class*="_channel_"]')
+				?.querySelector('[class*="_title_"]')?.textContent ?? "Unknown Radio"; // which radio we're tuning right now
 		state =
-			document.querySelector("._subtitle_165lg_43")?.textContent ??
+			document.querySelector('[class*="_subtitle_"]')?.textContent ??
 			"Unknown State/Country"; // the state where the radio serves
 
-		const elapsed = Math.floor(Date.now() / 1000),
-			presenceData: PresenceData = {
-				details,
-				state,
-				largeImageKey:
-					"https://cdn.rcd.gg/PreMiD/websites/R/Radio%20Garden/assets/logo.png",
-				startTimestamp: elapsed,
-			};
+		const presenceData: PresenceData = {
+			details,
+			state,
+			largeImageKey:
+				"https://cdn.rcd.gg/PreMiD/websites/R/Radio%20Garden/assets/logo.png",
+			startTimestamp: elapsed,
+		};
 		currentState = document
-			.querySelectorAll("._control_oyndo_11._modPlay_oyndo_53")[0]
-			.getAttribute("aria-label"); // get current state of the "button", not the "player", except when loading
-		if (currentState.length > 0) {
+			.querySelector("[class*='_modPlay_']")
+			?.getAttribute("aria-label"); // get current state of the "button", not the "player", except when loading
+		if (currentState) {
 			switch (currentState) {
 				case "stop": {
 					// currently playing
-					presenceData.smallImageKey = "statusplay";
+					presenceData.smallImageKey = Assets.Play;
 					presenceData.smallImageText = "Playing";
 					presenceData.details = details;
 					presenceData.state = state;
 					presenceData.startTimestamp = elapsed;
-
+					presenceData.buttons = [
+						{ label: "Listen To The Radio", url: document.location.href },
```

Comment:
Let me know if I need to word this differently, cuz i wasnt sure lol

Reply:
Maybe just 'Listen'?
```suggestion
						{ label: "Listen", url: document.location.href },
```